src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -43,6 +43,11 @@ public sealed class PolicyContract : NativeContract
         /// </summary>
         public const uint DefaultAttributeFee = 0;
 
+        /// <summary>
+        /// The default fee for NotaryAssisted attribute.
+        /// </summary>
+        public const uint DefaultNotaryAssistedAttributeFee = 1000_0000;
```

Comment:
How is the fee decided?

Reply:
As everything in Policy, it's controlled by committee. We can probably have the default of zero for some symmetry with other attributes (and set something real after deploy).

Reply:
According to https://github.com/neo-project/neo/issues/1573#issuecomment-704874472:
> The FEE is expected to be low enough for that to not a problem for regular users, but big enough to prevent spamming the network with service requests.

The precise calculations are presented here: https://github.com/nspcc-dev/neo-go/issues/1545#issue-743844176.

Note, that this value is customizable via native Policy (using the same way like for other attributes).

Reply:
> We can probably have the default of zero for some symmetry with other attributes (and set something real after deploy).

Missed the comment. Set to 0 for now? How about NeoFS chains?

Reply:
That's a suggestion if we have a problem with the default value. Balances are gonna be different in FS chains (but no one care about balances there), but I think transactions should work the same. But again, only if there are objections for the non-zero default.